mypy/main.py

```diff
@@ -677,6 +671,12 @@ def add_invertible_flag(
         help="Disallow usage of generic types that do not specify explicit type parameters",
         group=disallow_any_group,
     )
+    add_invertible_flag(
```

Comment:
I moved this down ~20 lines or so, as all invertible flags for this group were defined after non-invertible flags prior to now and I wanted to maintain that convention (assuming this was intentional and not just a coincidence)